portfolio/src/main/webapp/scripts/toggle.js

```diff
@@ -0,0 +1,32 @@
+let isShowingSection = false;
+let activeSection = null;
+
+/**
+ * Decides wether to load or hide the section.
+ */
+function toggleSection(section){
+  if(isShowingSection) (hideSection());
```

Comment:
[Optional] It isn't explicitly required, but I strongly recommend against skipping the braces for an if statement.

Consider the following code

if (buttonWasClicked) (doStuff());

Suppose I want to edit this to also do some other stuff by calling doOtherStuff().  It is easy for someone to make the following edit

if (buttonWasClicked)
  doStuff();
  doOtherStuff();

However, this new code won't function as expected.  The second function will run regardless of whether or not the button was clicked.  Thus, it is safer to include braces, even when only one line is needed.